css/index.css

```diff
@@ -127,3 +127,93 @@ table {
   border-collapse: collapse;
   border-spacing: 0;
 }
+/* Coding Starts here */
+
+container{
+  width: 800px;
+  margin: auto;
+}
+ /* NO CSS for head!! */
+
+.body{
+  align-items: center;
+  max-width: 100px;
+  margin: 0 auto;
+  background-color: azure;
+}
+
+.nav{
+  flex-direction: row;
+  justify-content: flex-end;
+  font-size: 20px;
+
+  
+}
+.header{
```

Comment:
`.header` is a class. Looks like this: `<div class='header'>`
`#header` is an id. Looks like this: `<div id='header'>`
`header` is an element or, aka, an html tag. Looks like this: `<header>`

In your html, you have a header tag with an `h1` and a paragraph. On your site, it's not styled at all. This is because the css needs to look like this:

```
header {
    your styles go here
}
```